Makefile

```diff
@@ -334,6 +337,7 @@ install: $(build_depsbindir)/stringreplace $(BUILDROOT)/doc/_build/html
 	done
 
 	$(INSTALL_M) $(build_bindir)/julia* $(DESTDIR)$(bindir)/
+	$(INSTALL_M) $(build_bindir)/embedding* $(DESTDIR)$(bindir)/
```

Comment:
I'll take a look at `contrib/build_sysimg.jl` for ideas on how to do this. Thanks for the input, @tkelman, @yuyichao .